src/commands/developers/restart.js

```diff
@@ -0,0 +1,23 @@
+const { CommandStructures, SwitchbladeEmbed } = require('../../')
+const { Command, CommandRequirements } = CommandStructures
+
+module.exports = class Restart extends Command {
+  constructor(client) {
+    super(client)
+    this.name = 'restart'
+    this.aliases = ['rs']
+    this.category = 'developers'
+    this.hidden = true
+
+    this.requirements = new CommandRequirements(this, { devOnly: true })
+  }
+
+  async run({ message }) {
+    const embed = new SwitchbladeEmbed();
+
+    message.channel.send(embed
+      .setTitle('Service Restart')
+      .setDescription('Restarting...')
+    ).then(message.client.destroy()).then(message.client.login());

```

Comment:
This wont work, you need to use `.then(() => message.client.destroy())` and please no `;`. fucking run `npm run test`